CONVERT_SERVICES/src/controller/apis/endpointconverter.py

```diff
@@ -33,4 +33,11 @@ def getRequest(self):
         return self.request
 
     def Send_File(self, dir_output, file_name):
```

Comment:
here if yo can change the function name to send_file in lowercase, uppercase it's for classes